skytemple_files/common/ppmdu_config/script_data.py

```diff
@@ -364,6 +391,14 @@ def __init__(
         self.op_codes = op_codes
         self.ground_state_structs = ground_state_structs
 
+    def _insert_constants_with_prefix(self, vars: Sequence[ScriptDataConstant], prefix: str) -> None:
```

Comment:
This is technically not entirely OK, since the setters would insert new constants, ie. for new actors or levels but not remove removed or renamed ones. Any idea what we could do about that?